### PR TITLE
Port str case commands

### PR DESCRIPTION
**Pull request by onthebridgetonowhere**
Still need to support the column paths. I guess maybe we should figure how to do this soon :)

### CODE REVIEW

**Comment by jntrnr** [comment]
Agreed, will see if we can add that first, so we don't have so many Todos 😅

**Comment by jntrnr** [comment]
Taking a look now

**Comment by jntrnr** [comment]
@onthebridgetonowhere - okay, I added some support for paths to columns: https://github.com/nushell/engine-q/pull/290

**Comment by onthebridgetonowhere** [comment]
Thanks, I'll have a look!

**Comment by jntrnr** [comment]
This looks great! Can you also throw a test in the sec/tests.rs file (or as an example to a command) that tests the optional cell path?

**Comment by onthebridgetonowhere** [comment]
Added a `cell path` test for each `str case` command in `src/tests.rs`. Should be good to go now :-)

File: src/tests.rs [hunk 1]
```diff
@@ -849,3 +849,43 @@ fn range_and_reduction() -> TestResult {
...
+    run_test(
+        r#"([[lang, gems]; [nuTest, 100]]).lang | get 0 | str snake-case"#,
+        "nu_test",
+    )
+}
+
+#[test]
+fn str_case_screaming_snake_cell_path() -> TestResult {
+    run_test(
+        r#"([[lang, gems]; [nuTest, 100]]).lang | get 0 | str screaming-snake-case"#,
```
Comment (jntrnr) [inline src/tests.rs:888]: Thanks for adding some tests here. Can we change some of these to take in a column path? These are using the default but not taking in an argument that gives the column to update.

Comment (onthebridgetonowhere) [reply src/tests.rs:888]: Oh, I see, I was not sure how to do this. Thanks for checking and clarifying it!

Comment (onthebridgetonowhere) [reply src/tests.rs]: Fixed it!

**Comment by jntrnr** [comment]
looks good!

**Pull request merged**